chromium_src/chrome/browser/download/download_target_determiner.cc

```diff
@@ -0,0 +1,13 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+// Prompting the user for download location shouldn't be a factor in determining
```

Comment:
I'm confused because this looks like an upstream issue if there is an issue here, but the location does affect the danger level which is why macOS prompts for permission to access different file system locations for each app